content/cloud-docs/users-teams-organizations/single-sign-on/index.mdx

```diff
@@ -63,12 +63,14 @@ If an organization's owners disable SSO (or downgrade the organization's account
 
 ## Managing Team Membership Through SSO
 
-Terraform Cloud can automatically add users to teams based on their SAML assertion, so you can manage team membership in your directory service. To do this, you must specify the `MemberOf` SAML attribute, and make sure the AttributeStatement in the SAMLResponse contains a list of AttributeValue items in the correct format (i.e., comma-separated list of team names).
+Terraform Cloud can automatically add users to teams based on their SAML assertion, so you can manage team membership in your directory service. To do this, you must specify the `MemberOf` SAML attribute, and make sure the AttributeStatement in the SAMLResponse contains a list of AttributeValue items in the correct format (i.e., comma-separated list of team names or SSO team IDs).
 
 If the SAML assertion includes a `MemberOf` attribute, users logging in via SSO are automatically added to the teams included in their assertion and removed from any teams that _aren't_ included in their assertion. This overrides any manually set team memberships; whenever the user logs in, their team membership is adjusted to match their SAML assertion.
 
 Any team names that don't match existing teams are ignored; Terraform Cloud will not automatically create new teams. Terraform Cloud expects the team names in the `MemberOf` SAML attribute to exactly match its own team names. This match is case sensitive. It is not possible to assign users to the `owners` team through this attribute.
 
+In some circumstances the `MemberOf` SAML attribute may not be human readable, due to restrictions in your SSO provider for example. In this case, in order to maintain human readalbe team names in Terraform Cloud, you may set an SSO Team ID for the team which matches the SAML attribute instead.
```

Comment:
I thought to myself this morning that it might be nice to have a paragraph explaining _why_ SSO Team IDs are a thing, so I added a blurb here. I thought I could get it in before you re-reviewed @laurapacilio but no luck :(

Would you mind proofing this line to make sure you still approve?